source/addonStore/dataManager.py

```diff
@@ -176,13 +176,14 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 		try:
 			with open(cacheFilePath, 'r', encoding='utf-8') as cacheFile:
 				cacheData = json.load(cacheFile)
-		except Exception:
+		except (KeyError, JSONDecodeError):

```

Comment:
No, I think the bare `except` was okay here (back out that change). I was referring to guarding the `json.loads` call in `_createStoreCollectionFromJson` itself with a `try`/`except`.

Reply:
I don't think that's necessary, an error being raised outside of loading the cached data from file on startup will fail safely.